.devcontainer/Dockerfile

```diff
@@ -28,7 +28,12 @@ RUN useradd dev -u 1000 -m -s /bin/bash && \
 USER dev
 
 
+# Install GEF for sanity & its dependencies
+RUN sudo apt install -y file && bash -c "$(wget https://gef.blah.cat/sh -O -)"
```

Comment:
I don't use VS, and I am not responsible for this repo, but I know a `curl | sh` when I see one